.circleci/config.yml

```diff
@@ -98,7 +98,27 @@ commands:
             ./get_helm.sh --version "${HELM_VERSION}"
             rm ./get_helm.sh
             helm version --short
+  tf_plan_test:
+    steps:
+      - aws-cli/setup:
+         role-arn: "${AWS_IAM_ROLE}"
+         session-duration: "900"
+      - run:
+          name: Terraform plan test
+          working_directory: examples
+          command: |
+            bash tf-plan-test.sh
```

Comment:
if you require bash, why isn't it a shebang in the tf-plan-test.sh?

Reply:
It is https://github.com/dominodatalab/terraform-aws-eks/blob/5896cf2bd812b758d921f004776d564e55df1251/examples/tf-plan-test.sh#L1.

Reply:
so why bash it?